Rex.podspec

```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '9.0'
 
   s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => s.version }
```

Comment:
Technically I'm more or less still the author. We could add others since _I think_ that supports a list of authors.